api_client/python/timesketch_api_client/client.py

```diff
@@ -801,6 +801,30 @@ def store_aggregation(
         _ = self.lazyload_data(refresh_cache=True)
         return self.get_aggregation(objects[0].get('id'))
 
+    def comment_event(self, event_id, index, comment_text):
+        """
+        Adds a comment to a single event.
+
+        Args:
+            event_id: id of the event
+            index: The Elasticsearch index name
+            comment_text: text to add as a comment
+        Returns:
+             a json data of the query.
+        """
+        form_data = {
+            u'annotation': comment_text,
```

Comment:
don't have the u'' just use '' (this is py3 code and we've imported unicode_literals from future for py2 compatibility)

Reply:
should be good now